DataAnalysis.md

```diff
@@ -0,0 +1,36 @@
+# Analysing Model Results
```

Comment:
Do we want to divide into beginner and advanced skill sets like Claire was doing?